reptiles.html

```diff
@@ -27,6 +26,7 @@
     <section class="section-animals">
       <div class="left-side">
         <div class="about-animal">
+          <h1 class="about-us-header">About Reptiles</h1>
```

Comment:
this h1 doesn't need a class and should be inside of a div with a class of "about-us-header"

Reply:
check out the index.html to confirm